CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.6.3)
+project(kvssdp C)
+
+include(sdpFilePaths.cmake)
+
+include_directories(${SDP_INCLUDE_PUBLIC_DIRS})
```

Comment:
I am not sure, but curious why not use target_include_directories

Reply:
Update to target_include_directories.